crypto/cms/cms_pwri.c

```diff
@@ -63,9 +63,6 @@ CMS_RecipientInfo *CMS_add0_recipient_password(CMS_ContentInfo *cms,
     if (wrap_nid <= 0)
         wrap_nid = NID_id_alg_PWRI_KEK;
 
-    if (pbe_nid <= 0)
```

Comment:
This is the only use of the `pbe_nid` argument to the `CMS_add0_recipient_password()` function - how come the function has an unused argument?